Solutions/PCMatic SuperShield/Data/Solution_PCMaticSuperShield.json

```diff
@@ -0,0 +1,27 @@
+{
+    "Name": "PCMatic SuperShield",
+    "Author": "John Farley - john@pcmatic.com",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/johnpcmatic/Azure-Sentinel/pcm-sentinel/Solutions/PCMatic%20SuperShield/Logo/pcmatic-green-outline.png\" width=\"200px\" height=\"45px\">",
```

Comment:
@johnpcmatic , please use the raw libk of svg, and also check for the dimensions [here](https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Create-Azure-Sentinel-Solution/V2). 
![image](https://user-images.githubusercontent.com/94349919/207875208-7e166909-b962-4a86-858a-8939ea89e626.png)